gremlin-python/src/main/python/gremlin_python/driver/driver_remote_connection.py

```diff
@@ -48,21 +67,25 @@ def __init__(self, url, traversal_source = "g", protocol_factory=None,
                                      password=password,
                                      kerberized_service=kerberized_service,
                                      headers=headers,
+                                     session=session,
                                      **transport_kwargs)
         self._url = self._client._url
         self._traversal_source = self._client._traversal_source
 
     def close(self):
+        logging.info("closing DriverRemoteConnection with url '%s'", str(self._url))
         self._client.close()
 
     def submit(self, bytecode):
+        logging.debug("submit with bytecode '%s'", str(bytecode))
         result_set = self._client.submit(bytecode, request_options=self._extract_request_options(bytecode))
         results = result_set.all().result()
         return RemoteTraversal(iter(results))
 
-    def submitAsync(self, bytecode):
+    def submit_async(self, bytecode):
```

Comment:
i asked you to keep the old `submitAsync` form in `Client` as it was a breaking change. not sure that it matter so much here because I don't think anyone calls this directly. maybe just to be safe, retain the old form in addition to the new and we'll remove the old on merge to `master`.

Reply:
Sounds good, will do.